rules/cloud/aws_ec2_vm_export_failure.yml

```diff
@@ -0,0 +1,28 @@
+title: AWS EC2 VM Export Failure
+id: 54b9a76a-3c71-4673-b4b3-2edb4566ea7b
+status: experimental
+description: An attempt to export an AWS EC2 instance has been detected. A VM Export might indicate an attempt to extract information from an instance.
+references:
+  - https://docs.aws.amazon.com/vm-import/latest/userguide/vmexport.html#export-instance     
+author: Diogo Braz
+date: 2020/04/16
+tags:
+  - attack.collection    
+  - attack.t1005      
+  - attack.exfiltration
+  - attack.t1537
+level: low
+logsource:
+  service: cloudtrail
+detection:
+  selection:           
+    eventName: 'CreateInstanceExportTask'
+    eventSource: 'ec2.amazonaws.com'
+  filter1:
+    errorMessage: '*'
```

Comment:
@thomaspatzke : is this '*' correct or do we have to use 'null' and negate it in the condition?

Reply:
That's ok